Index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Page Title</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.2.1/css/fontawesome.min.css" integrity="sha384-QYIZto+st3yW+o8+5OHfT6S482Zsvz2WfOzpFSXMF9zqeLcFV0/wlZpMtyFcZALm" crossorigin="anonymous">
+</head>
+<body> 
+    <div class="wrapper">
+      <section class="section1">
+       <header class="toolbar">           
+            <span id="logo">My Logo</span> 
+            <span id="switch">               
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>             
+            </span>                  
```

Comment:
The hamburger button on the right, should have a `nav` tag.
The `nav` tag is for semantics reasons and the right position for convention and user experience for mobile devices.